### PR TITLE
[zh] Add the new tasks page of 'Customizing Istio Metrics'

### PR DESCRIPTION
**Pull request by ydFu**
*  Sync with English version of 'Customizing Istio Metrics'.

Signed-off-by: ydFu <ader.ydfu@gmail.com>

Please provide a description for what this PR is for.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/tasks/observability/metrics/customize-metrics/index.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+* [Grafana](/zh/docs/tasks/observability/metrics/using-istio-dashboard/)
+* [Kiali](/zh/docs/tasks/observability/kiali/)
+* [Prometheus](/zh/docs/tasks/observability/metrics/querying-metrics/)
+
+默认情况下，Istio 定义并生成一组标准指标（例如： `requests_total`），但您也可以自定义它们并创建新指标。
+
+## 自定义统计配置{#custom-statistics-configuration}
+
+Istio 使用 Envoy 代理生成指标并在 `EnvoyFilter` 中
+[manifests/charts/istio-control/istio-discovery/templates/telemetryv2_{{< istio_version >}}.yaml]({{<github_blob>}}/manifests/charts/istio-control/istio-discovery/templates/telemetryv2_{{< istio_version >}}.yaml)
```
Comment (windsonsea) [inline content/zh/docs/tasks/observability/metrics/customize-metrics/index.md:24]:
```suggestion
[manifests/charts/istio-control/istio-discovery/templates/telemetryv2_{{< istio_version >}}.yaml]({{<github_blob>}}/manifests/charts/istio-control/istio-discovery/templates/telemetryv2_{{< istio_version >}}.yaml) 文件的 `EnvoyFilter` 中提供配置。
```

File: content/zh/docs/tasks/observability/metrics/customize-metrics/index.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+* [Grafana](/zh/docs/tasks/observability/metrics/using-istio-dashboard/)
+* [Kiali](/zh/docs/tasks/observability/kiali/)
+* [Prometheus](/zh/docs/tasks/observability/metrics/querying-metrics/)
+
+默认情况下，Istio 定义并生成一组标准指标（例如： `requests_total`），但您也可以自定义它们并创建新指标。
+
+## 自定义统计配置{#custom-statistics-configuration}
+
+Istio 使用 Envoy 代理生成指标并在 `EnvoyFilter` 中
```
Comment (windsonsea) [inline content/zh/docs/tasks/observability/metrics/customize-metrics/index.md:23]:
```suggestion
Istio 使用 Envoy 代理生成指标并在
```

File: content/zh/docs/tasks/observability/metrics/customize-metrics/index.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+[manifests/charts/istio-control/istio-discovery/templates/telemetryv2_{{< istio_version >}}.yaml]({{<github_blob>}}/manifests/charts/istio-control/istio-discovery/templates/telemetryv2_{{< istio_version >}}.yaml)
+
+配置自定义统计信息涉及两个部分 `EnvoyFilter`： `definitions` 和 `metrics`。
```
Comment (windsonsea) [inline content/zh/docs/tasks/observability/metrics/customize-metrics/index.md:26]:
```suggestion
配置自定义统计信息涉及 `EnvoyFilter` 的两个部分： `definitions`  和  `metrics`。
```

File: content/zh/docs/tasks/observability/metrics/customize-metrics/index.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+在 `definitions` 的部分支持用名字建立新的指标、正规表达式和各种指标类型（`counter` 、 `gauge` 和 `histogram`）。
```
Comment (windsonsea) [inline content/zh/docs/tasks/observability/metrics/customize-metrics/index.md:27]:
```suggestion
在 `definitions` 部分支持用名字、期望值表达式和指标类型（`counter` 、 `gauge` 和 `histogram`）创建新的指标。
```

**Review by windsonsea — CHANGES_REQUESTED** [review]
may need to check more about the localized text.

**Comment by ydFu** [comment]
@windsonsea 
Thank for your code view.

File: content/zh/docs/tasks/observability/metrics/customize-metrics/index.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+与 Mixer 表达式语言不同，不支持 pipe (`|`) 运算符，但您
+可以使用 `has` 或 `in` 操作符来模拟它，例如：
+
+{{< text plain >}}
+has(request.host) ? request.host : "unknown"
+{{< /text >}}
+
+有关详细信息，请参阅[通用表达式语言](https://opensource.google/projects/cel)。
+
+Istio 公开了所有标准[Envoy 属性](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/advanced/attributes)。
```
Comment (Xunzhuo) [inline content/zh/docs/tasks/observability/metrics/customize-metrics/index.md:168]: Need a space between en/zh letters

**Review by Xunzhuo — CHANGES_REQUESTED** [review]
Minor comment

**Review by Xunzhuo — APPROVED** [review]
Looks good, thanks @ydFu

**Pull request merged**